content/other/molecules-amino-acids/ala/dataset.json

```diff
@@ -0,0 +1,50 @@
+{
+    "slug": "C3H7NO2",
+    "class": {
+        "slug": "ala",
+        "name": "alanine",
+        "attributeList": [
+            {
+                "name": "attribute1",
+                "pythonType": "type",
+                "doc": "Description of attribute1"
+            },
+            {
+                "name": "attribute2",
+                "pythonType": "type",
+                "doc": "Description of attribute2"
+            }
+        ]
+    },
+    "collection": {
+        "$path": "/other/molecules-amino-acids/_meta/collection.json"
+    },
+    "data": [
+        {
+            "dataUrl": "https://datasets.cloud.pennylane.ai/datasets/h5/other/AminoAcids/ala.h5",
+            "parameters": {
+                "name": "alanine"
+            },
+            "extra": {}
+        }
+    ],
+    "downloadName": "alanine",
+    "features": [
+        {
+            "slug": "dataset-attributes",
+            "title": "Dataset Attributes",
+            "type": "DATA",
+            "content": {
+                "$path": "features/dataset-attributes.md"
```

Comment:
Deploying raises an error because there is not `features/dataset-attributes.md`
If the data table will be the same for all of the datasets, we can create one file and rewrite these paths to all point to that file.

Reply:
Just a reminder about this.